src/mscorlib/shared/System/Diagnostics/StackFrame.cs

```diff
@@ -0,0 +1,272 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System.Text;
+using System;
+using System.IO;
+using System.Reflection;
+using System.Diagnostics.Contracts;
+
+namespace System.Diagnostics
+{
+    /// <summary>
+    /// There is no good reason for the methods of this class to be virtual.
```

Comment:
No idea. I would think so, I don't see any reasons to derive from this class and I don't think anyone does. I'd prefer to leave this discussion for the CoreCLR integration phase though.